src/gdrdrv/Makefile

```diff
@@ -18,7 +18,10 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-NVIDIA_SRC_DIR ?= $(shell find /usr/src/nvidia-* -name "nv-p2p.h"|head -1|xargs dirname || echo "NVIDIA_DRIVER_MISSING")
+NVIDIA_SRC_DIR ?= $(shell d=/usr/src; f=''; \
```

Comment:
The code is getting more complex. How about this?
```
NVIDIA_SRC_DIR ?= $(shell (find /usr/src/kernel-modules/nvidia-* /usr/src/nvidia-* -name "nv-p2p.h" -print -quit |head -1|xargs dirname || echo "NVIDIA_DRIVER_MISSING") 2>/dev/null )
```

Reply:
That's what I had first, I found it ugly as `find` will most likely report an error (which gets supressed). Trying to find a way around this eventually lead to a slightly over-engineered  variant :/ 
With `-print -quit` it should be possible to drop the `head -1` though.